lib/vp.js

```diff
@@ -577,6 +577,15 @@ class VP {
             } else {
               shortTitle = shortTitlePrefix;
             }
+            if (openingDate && documentType === DOCUMENT_TYPES.DECREET) {
+              const dateOptions = {
+                year: 'numeric',
+                month: 'long',
+                day: 'numeric',
+              };
+              const dateSuffix = openingDate.toLocaleDateString('nl-BE', dateOptions)
+              shortTitle += ", aagenomen door het Vlaams Parlement op " + dateSuffix;
+            }
```

Comment:
Ticket mentions doing it only for Decreet.
However on PROD I noticed that it is also used in Motie/Resolutie but in the longTitle.
Since we don't fill in the longTitle, the users must do it themselves but copy pasting the shorttitle in the longTitle en making a new shorttitle (speculation)
In that case, having that for all types means less typing by the users.

If this is eventually not wanted by the users, we will make it Decreet only again.

Reply:
made it work for all types